src/main/java/com/flipkart/foxtrot/client/Document.java

```diff
@@ -0,0 +1,55 @@
+package com.flipkart.foxtrot.client;
```

Comment:
foxtrot-common is not pushed as a module to artifactory. This is just a placeholder.